utils/src/check/clobber-functions.c

```diff
@@ -140,6 +140,57 @@ static struct clobber_function clobber_pb_qconf_wrong_version = {
 	&do_clobber_pb_qconf_wrong_version,
 };
 
+/*
+ * Pick a random non-empty quorum slot then returns a pointer to that slot
+ */
+static struct scoutfs_quorum_slot *pick_random_qslot(void)
+{
+	int j = 0;
+	int filled_slots[SCOUTFS_QUORUM_MAX_SLOTS];
+	int rand_slot;
+	uint16_t family;
+	srandom(time(NULL));
+
+	for (int i = 0; i < SCOUTFS_QUORUM_MAX_SLOTS; i++) {
```

Comment:
This won't compile on `el7` because `for (int i` isn't allowed just yet there. You'll have to declare `int i;` above at the top of the function.

Reply:
okay, I did this for indexing the slots in super.c as well, should I change it there to?

Reply:
Yes.